### PR TITLE
docs: Clarify strlist_split

### PR DESCRIPTION
**Pull request by CodeYan01**
### Description

image

### Motivation and Context

Saw it, and had to debug to figure out its behavior. Also had to search OBS repo to find out how to use it.

### How Has This Been Tested?

Built and viewed locally. Also ran the example code.

### Types of changes

- Documentation (a change to documentation pages)

### Checklist:

- [x] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [x] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [x] My code is not on the master branch.
- [x] The code has been tested.
- [x] All commit messages are properly formatted and commits squashed where appropriate.
- [x] I have included updates to all appropriate documentation.

### CODE REVIEW

File: docs/sphinx/reference-libobs-util-dstr.rst [hunk 1]
```diff
@@ -87,8 +87,29 @@ General String Helper Functions
 
 .. function:: char **strlist_split(const char *str, char split_ch, bool include_empty)
 
-   Splits a string in to a list of multiple sub-strings.  Free with
-   :c:func:`strlist_free()`.
+   Splits a string in to a list of multiple sub-strings, terminated by
+   ``NULL``. If ``split_ch`` does not exist in the string, the first
+   sub-string will be the same as ``str``. Free with :c:func:`strlist_free()`.
+
```
Comment (WizardCM) [inline docs/sphinx/reference-libobs-util-dstr.rst:93]: Please document the other two parameters using the `:param:` format too.

Comment (CodeYan01) [reply docs/sphinx/reference-libobs-util-dstr.rst:93]: Updated, please check

**Comment by derrod** [comment]
This PR modifies submodules, please fix that.

**Comment by CodeYan01** [comment]
Ooops, fixed.

**Pull request merged**